sql/plan/show_create_table_test.go

```diff
@@ -55,8 +55,8 @@ func TestShowCreateTable(t *testing.T) {
 	expected := sql.NewRow(
 		table.Name(),
 		"CREATE TABLE `test-table` (\n  `baz` text NOT NULL,\n"+
-			"  `zab` int DEFAULT 0,\n"+
-			"  `bza` bigint unsigned DEFAULT 0 COMMENT 'hello',\n"+
+			"  `zab` int DEFAULT '0',\n"+
```

Comment:
MySQL allows unquoted numeric literals. Are we still able to keep that behavior?

The numeric default value is still quoted in the `show create table` output, but MySQL does allow it as unquoted for initial input. 

It would be good to make sure we still have a test covering this case.